src/services/DavClient.js

```diff
@@ -26,8 +26,8 @@ import { getRequestToken } from '@nextcloud/auth'
 import { generateRemoteUrl } from '@nextcloud/router'
 
 // init webdav client on default dav endpoint
-const client = createClient(generateRemoteUrl('dav'),
-	{ headers: { requesttoken: getRequestToken() || '' } },
-)
-
-export default client
+export const getDavClient = () => {
```

Comment:
Why get when its creating? :P

Reply:
To not mix with `import { createClient } from 'webdav'`?

Though it could be renamed, I think that using a `get` prefix is fine here. We're expecting from it to return an object, not to 'create and return'. It might be (and should be) further optimized to not re-create a client each time, unless `requestToken` has ben changed

Reply:
Agree with Joas, that `get` prefix is usually used for getting something existed, not creating. Alternatives could be
- `buildDavClient`
- `makeDavClient`

But fine to me for consistency with other apps.